std/process.d

```diff
@@ -3155,11 +3163,17 @@ unittest
     assertThrown(environment["std_process"]);
 
     // get() without default value
-    assert (environment.get("std_process") == null);
+    assert (environment.get("std_process") is null);
 
     // get() with default value
     assert (environment.get("std_process", "baz") == "baz");
 
+    version(Posix) {
```

Comment:
there's a comment on the previous version of this (#3903) that says:

> Windows command prompt treats empty environment variables the same as non-existent 

Given that, we should probably not differentiate.